CMake Error finding lib FLANN when using PCL in a project (ROS Jazzy)

```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/pcl/PCLConfig.cmake:64 (message):
  search is required but flann was not found
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/pcl/PCLConfig.cmake:396 (pcl_report_not_found)
  /usr/lib/x86_64-linux-gnu/cmake/pcl/PCLConfig.cmake:588 (find_external_library)
  /opt/ros/jazzy/share/pcl_conversions/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)
  /opt/ros/jazzy/share/pcl_conversions/cmake/pcl_conversionsConfig.cmake:41 (include)
  /opt/ros/jazzy/share/pcl_ros/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)
  /opt/ros/jazzy/share/pcl_ros/cmake/pcl_rosConfig.cmake:41 (include)
  /home/rein/dev/autoware/install/share/autoware_point_types/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)
  /home/rein/dev/autoware/install/share/autoware_point_types/cmake/autoware_point_typesConfig.cmake:41 (include)
  /opt/ros/jazzy/share/ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake:67 (find_package)
  /opt/ros/jazzy/share/autoware_cmake/cmake/autoware_package.cmake:49 (ament_auto_find_build_dependencies)
  CMakeLists.txt:5 (autoware_package)
```

--- a/cmake/Modules/FindFLANN.cmake
+++ b/cmake/Modules/FindFLANN.cmake
@@ -37,6 +37,7 @@
 # Early return if FLANN target is already defined. This makes it safe to run
 # this script multiple times.
 if(TARGET FLANN::FLANN)
+    set(FLANN_FOUND ON)
     return()
 endif()
 

